plugins/modules/helm.py

```diff
@@ -676,6 +699,24 @@ def helmdiff_check(
     if reuse_values:
         cmd += " --reuse-values"
 
+    if reset_then_reuse_values:
+        helm_diff_version = get_plugin_version("diff")
+        helm_version = module.get_helm_version()
+        if LooseVersion(helm_diff_version) < LooseVersion("3.9.12"):
```

Comment:
I see a case when both requirement aren't meet, the only fist check will be executed and only `reset_then_reuse_values requires helm diff >= 3.9.12, current version is {0}` will be on fail message. 

So, it better to check both conditions and have a message that contains helm, helm diff or both, depends on versions installed.

If you don't mind, I will prepare commit that improve this part